library/src/main/java/com/datatorrent/lib/db/jdbc/AbstractJdbcPollInputOperator.java

```diff
@@ -269,20 +253,38 @@ public void deactivate()
     store.disconnect();
   }
 
+  protected void insertResultsInQueue(PreparedStatement preparedStatement) throws SQLException, InterruptedException
```

Comment:
The function name is misleading as you are also running the DB query in this function.